### PR TITLE
[Azure Pipelines] Run scheduled runs of Safari stable

### PR DESCRIPTION
**Pull request by foolip**
This requires fixing a small issue with how the --browser-version
argument is used, to make it possible to override.

### CODE REVIEW

**Comment by foolip** [comment]
https://dev.azure.com/web-platform-tests/wpt/_build/results?buildId=11343 is a scheduled run from this branch.

**Comment by foolip** [comment]
The Azure Pipelines failures are because of https://github.com/web-platform-tests/wpt/issues/16111, the interesting part here will be whether the full runs work and show up on wpt.fyi.

**Comment by foolip** [comment]
It worked:
https://wpt.fyi/results/?run_id=5639692737314816&run_id=4811294142103552
https://staging.wpt.fyi/results/?run_id=5135624860336128&run_id=6290544921673728

File: .azure-pipelines.yml [hunk 1]
```diff
@@ -198,6 +198,40 @@ jobs:
...
+  - template: tools/ci/azure/install_certs.yml
+  - template: tools/ci/azure/install_safari.yml
+    parameters:
+      channel: stable
+  - template: tools/ci/azure/update_hosts.yml
+  - template: tools/ci/azure/update_manifest.yml
+  - script: |
+      # TODO: drop this when `safaridriver --version` is supported.
+      SAFARI_VERSION="$(/usr/libexec/PlistBuddy -c 'Print :CFBundleShortVersionString' /Applications/Safari.app/Contents/Info.plist) ($(/usr/libexec/PlistBuddy -c 'Print :CFBundleVersion' /Applications/Safari.app/Contents/Info.plist))"
+      no_proxy='*' ./wpt run --no-manifest-update --no-restart-on-unexpected --no-fail-on-unexpected --this-chunk=$(System.JobPositionInPhase) --total-chunks=$(System.TotalJobsInPhase) --chunk-type hash --log-wptreport $(Build.ArtifactStagingDirectory)/wpt_report_$(System.JobPositionInPhase).json --browser-version "$SAFARI_VERSION" safari
```
Comment (Hexcles) [inline .azure-pipelines.yml]:
```suggestion
      no_proxy='*' ./wpt run --no-manifest-update --no-restart-on-unexpected --no-fail-on-unexpected --this-chunk=$(System.JobPositionInPhase) --total-chunks=$(System.TotalJobsInPhase) --chunk-type hash --log-wptreport $(Build.ArtifactStagingDirectory)/wpt_report_$(System.JobPositionInPhase).json --log-wptscreenshot $(Build.ArtifactStagingDirectory)/wpt_screenshot_$(System.JobPositionInPhase).txt --browser-version "$SAFARI_VERSION" safari
```

**Review by gsnedders — APPROVED** [review]

**Comment by foolip** [comment]
@jgraham do you know what's up with the total Taskcluster failure in https://tools.taskcluster.net/groups/ZgIDZj0dT0uhji7oPAX82w? I'm not seeing it on all PRs (phew) so I'll rebase and see what happens, but if you don't know what this is, can you take a look and maybe file an issue?

**Comment by Hexcles** [comment]
@foolip you need to rebase and force-push the branch.

**Comment by foolip** [comment]
@Hexcles that's what I did :)

**Comment by Hexcles** [comment]
This is because of https://bugzilla.mozilla.org/show_bug.cgi?id=1539523

**Comment by foolip** [comment]
Remaining failures are https://github.com/web-platform-tests/wpt/issues/16111.

**Pull request merged**